activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -506,6 +506,8 @@ def drop_table(table_name, options = {})
 
       # Add a new +type+ column named +column_name+ to +table_name+.
       #
+      # See {ActiveRecord::ConnectionAdapters::TableDefinition.column}[rdoc-ref:ActiveRecord::ConnectionAdapters::TableDefinition#column].
```

Comment:
Is there any benefit to link to the page for users who see the `add_column` doc?
https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-column

Reply:
The `column` method comes with some more powerful options, mainly, the `index: true` option. That's why I feel like when navigating the docs it's worth pointing to.